### ISSUE TITLE
Assertion failed! and invalid Buffer length when creating new Image from sharp buffer

### ISSUE THREAD

**Issue by eloops** [creatale/node-dv#26]
I'm trying to pass a TIFF file through [sharp](http://sharp.dimens.io/), outputting to a buffer and then creating a new Image with the output.

node 0.12.7 x86, on Windows 8.1 64-bit. DV compiled with vs2013 (through node-gyp install dv)

When I do a conversion to PNG and pass it through as a buffer:
```javascript
var fpath = __dirname + '\\in\\dd-000.tif'
sharp(fs.readFileSync(fpath)).png().toBuffer(function(err, data, info) {
  if (err) {
    throw err
  } else {
    console.log(data.length)
    var image = dv.Image('png', data)
  }
})
````
This results in this error popping up on screen:
![assertion](https://cloud.githubusercontent.com/assets/5370130/9551219/18405c82-4df3-11e5-971b-d51065757943.PNG)

Similarly, if I instead convert it to a raw and pass it through along with the width/height:
```javascript
var fpath = __dirname + '\\in\\dd-000.tif'
sharp(fs.readFileSync(fpath)).raw().toBuffer(function(err, data, info) {
  if (err) {
    throw err
  } else {
    console.log(data.length)
    var image = dv.Image('gray', data, info.width, info.height)
  }
})
````
It errors with:
````
E:\node\rb\index.js:74
    var image = dv.Image('gray', data, info.width, info.height)
                   ^
Error: invalid Buffer length
    at Error (native)
    at E:\node\rb\index.js:74:20
````

I feel like I'm missing something here?

**Comment by eloops** [comment]
I seem to have fixed this issue, I'm not entirely sure how I've managed to though. I tried to run `npm install --arch=ia32 --msvs_version=2013` (from the [installation](http://sharp.dimens.io/en/stable/install/#windows) instructions for sharp/Windows) and node-gyp recompiled dv.

I had also used [bluebird](https://github.com/petkaantonov/bluebird) to promisify `fs`, although I wasn't using it.

I can't really point to either of these as solid 'fixes' (after re-compiling I saw the error at least a few more times). 

Here's what my code looks like now:
```javascript
      sharp(fs.readFileSync(fpath)).png().toBuffer(function(err, data, info) {
        if (err) {
          throw err
        } else {
          var image = new dv.Image('png', data)
          var tesseract = new dv.Tesseract('eng', image)
          tesseract.pageSegMode = 'auto_only'
          var word_bbox = tesseract.findWords()
          var para_bbox = tesseract.findParagraphs()

        }
      })
````
The only change that I can see is using the `new` word to force the constructor.
